tpcx_bb/queries/q01/tpcx_bb_query_01_sql.py

```diff
@@ -40,14 +40,14 @@
 @benchmark(
     compute_result=cli_args["get_read_time"], dask_profile=cli_args["dask_profile"]
 )
-def read_tables(data_dir):
+def read_tables(data_dir, bc):
```

Comment:
Jumps out at me since we just updated the default config options to include `file_format: parquet`, whereas these are hardcoded.

I'm good with the hardcoding in the queries, but maybe we should remove the config to avoid confusion.

Reply:
I think eventually we may want to create a reader abstraction for blazingsql, but I think it's out of scope for this PR